install_config/persistent_storage/topics/glusterfs_storageclass.adoc

```diff
@@ -41,6 +42,10 @@ Volume Options] for more GlusterFS volume options.
 <9> Optional. The
 link:https://docs.gluster.org/en/v3/Quick-Start-Guide/Architecture/[type of
 volume] to use.
+<10> Optional. Enables custom volume name support using the following format:
+`<volumenameprefix>_<namespace>_<claimname>_UUID`. If you create a new PVC
+called `myclaim` in your project `project1` using this storageClass, the PVC
+name will be `custom-project1-myclaim-UUID`.
```

Comment:
s/the PVC name will be `custom-project1-myclaim-UUID`/the volume name will be `custom_project1_myclaim_UUID`/

Reply:
Thanks, updated!